### PR TITLE
Docs release notes aug 16 2024

### PR DESCRIPTION
**Pull request by paperclypse**

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-8-16-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### New docs
+
+* [Create "host not reporting" NRQL alert conditions](/docs/alerts/create-alert/create-alert-condition/create-nrql-host-not-reporting-conditions) describes how to set up alerting for a loss of signal.
+
+
+### Major changes
+
+* Improved our installation and configuration steps in [introduction to AWS Lambda monitoring](/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/introduction-lambda/#setup) to make it easier to see how your Lambda functions are performing in our UI.
+* We've got a new [end-of-life announcements](/eol/) section in our docs to make it easier for you to keep track of what's being end-of-lifed and when. It also includes an RSS feed for your convenience.
```
Comment (rhetoric101) [inline src/content/docs/release-notes/docs-release-notes/docs-8-16-2024.mdx:15]:
```suggestion
* We've got a new [End-of-life announcements](/eol/) section in our docs to make it easier for you to keep track of what's scheduled for end-of-life. It also includes an RSS feed for your convenience.
```

File: src/content/docs/release-notes/docs-release-notes/docs-8-16-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+### Minor changes
+
+* Updated [version compatibility information](/docs/infrastructure/host-integrations/host-integrations-list/cloudfoundry-integrations/vmware-tanzu-nginx-buildpack-integration/#requirements-compatibility) for our NGINX integration buildpack for VMware Tanzu.
+* Our [infrastructure agent](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#system) now supports Ubuntu version 24.04.x (LTS version).
```
Comment (rhetoric101) [inline src/content/docs/release-notes/docs-release-notes/docs-8-16-2024.mdx:20]: @paperclypse this link goes to log forwarding. The compatibility matrix for the infrastructure agent itself has a lower Ubuntu version: https://deploy-preview-18405--docs-website-netlify.netlify.app/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent/

What do you think?

**Review by rhetoric101 — COMMENTED** [review]
@paperclypse thanks for creating these release notes! I inserted a few editing suggestions and questions.

File: src/content/docs/release-notes/docs-release-notes/docs-8-16-2024.mdx (hunk 1 above)
Comment (paperclypse) [inline src/content/docs/release-notes/docs-release-notes/docs-8-16-2024.mdx:15]:
```suggestion
* We've got a new [end-of-life announcements](/eol/) section in our docs to make it easier for you to keep track of what's scheduled for end of life. It also includes an RSS feed for your convenience.
```

File: src/content/docs/release-notes/docs-release-notes/docs-8-16-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+### Major changes
+
+* Improved our installation and configuration steps in [introduction to AWS Lambda monitoring](/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/introduction-lambda/#setup) to make it easier to see how your Lambda functions are performing in our UI.
+* We've got a new [end-of-life announcements](/eol/) section in our docs to make it easier for you to keep track of what's scheduled for end of life. It also includes an RSS feed for your convenience.
+
+### Minor changes
+
+* Updated [version compatibility information](/docs/infrastructure/host-integrations/host-integrations-list/cloudfoundry-integrations/vmware-tanzu-nginx-buildpack-integration/#requirements-compatibility) for our NGINX integration buildpack for VMware Tanzu.
+* Our [infrastructure agent](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#system) now supports Ubuntu version 24.04.x (LTS version).
```
Comment (paperclypse) [inline src/content/docs/release-notes/docs-release-notes/docs-8-16-2024.mdx:20]:
```suggestion
* When forwarding logs with our [infrastructure agent](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#system), we now support Ubuntu version 24.04.x (LTS version).
```

**Review by rhetoric101 — APPROVED** [review]
@paperclypse these look good to me!

**Pull request merged**